requirements.txt

```diff
@@ -1,83 +1,83 @@
-absl-py>=0.1.12
-alabaster>=0.7.10
-appnope>=0.1.0
-astor>=0.6.2
-attrs>=17.4.0
-Babel>=2.5.3
-bleach>=1.5.0
-certifi>=2018.1.18
-chardet>=3.0.4
-cycler>=0.10.0
-decorator>=4.2.1
-docutils>=0.14
-entrypoints>=0.2.3
-future>=0.16.0
-gast>=0.2.0
-grpcio>=1.10.0
-html5lib>=0.9999999
-hyperopt>=0.1
-idna>=2.6
-imagesize>=1.0.0
-ipykernel>=4.8.2
-ipython>=6.2.1
-ipython-genutils>=0.2.0
-ipywidgets>=7.1.2
-jedi>=0.11.1
-Jinja2>=2.10
-jsonschema>=2.6.0
-jupyter>=1.0.0
-jupyter-client>=5.2.3
-jupyter-console>=5.2.0
-jupyter-core>=4.4.0
-Keras>=2.1.5
-kiwisolver>=1.0.1
-Markdown>=2.6.11
-MarkupSafe>=1.0
-matplotlib>=2.2.2
-mistune>=0.8.3
-more-itertools>=4.1.0
-nbconvert>=5.3.1
-nbformat>=4.4.0
-nbsphinx>=0.3.1
-networkx>=2.1
-nose>=1.3.7
-notebook>=5.4.1
-numpy>=1.14.2
-packaging>=17.1
-pandas>=0.22.0
-pandocfilters>=1.4.2
-parso>=0.1.1
-pexpect>=4.4.0
-pickleshare>=0.7.4
-pluggy>=0.6.0
-prompt-toolkit>=1.0.15
-protobuf>=3.5.2.post1
-ptyprocess>=0.5.2
-py>=1.5.3
-Pygments>=2.2.0
-pymongo>=3.6.1
-pyparsing>=2.2.0
-pytest>=3.5.0
-python-dateutil>=2.7.1
-pytz>=2018.3
-PyYAML>=3.12
-pyzmq>=17.0.0
-qtconsole>=4.3.1
-requests>=2.18.4
-scipy>=1.0.1
-Send2Trash>=1.5.0
-simplegeneric>=0.8.1
-six>=1.11.0
-snowballstemmer>=1.2.1
-Sphinx>=1.7.2
-sphinxcontrib-websupport>=1.0.1
-termcolor>=1.1.0
-terminado>=0.8.1
-testpath>=0.3.1
-tornado>=5.0.1
-traitlets>=4.3.2
-urllib3>=1.22
-wcwidth>=0.1.7
-webencodings>=0.5.1
-Werkzeug>=0.14.1
-widgetsnbextension>=3.1.4
+absl-py==0.1.12
```

Comment:
Don't require strict versions yet. It's required for production usage but not ideal for research.

Reply:
Will check that.